web/libs/editor/src/regions/TextAreaRegion.jsx

```diff
@@ -8,9 +8,10 @@ import Registry from "../core/Registry";
 import { TextAreaModel } from "../tags/control/TextArea/TextArea";
 import { guidGenerator } from "../core/Helpers";
 
-import styles from "./TextAreaRegion/TextAreaRegion.module.scss";
+import styles from "./TextAreaRegion/TextAreaRegion.scss";
```

Comment:
hmm - i just want to confirm - with the move away from this being a sass module - are other styles imported from here and used (like `styles.mark`) still working?

Reply:
What was the reason for the change to move away from SCSS module filename in this case?

Reply:
The styles are being applied properly. See below:

![image](https://github.com/user-attachments/assets/cf6d6611-24cd-419a-bf1a-6591220dd4e7)

Reply:
Was there an issue with the `TextAreaRegion.module.scss` filename as to the styles not applying correctly?